src/pages/iou/steps/MoneyRequestConfirmPage.js

```diff
@@ -272,6 +272,7 @@ function MoneyRequestConfirmPage(props) {
                     props.iou.tag,
                     reportID,
                     props.iou.merchant,
+                    false,
```

Comment:
The orders are not correct.
Should be `currency, merchant, category, tag, existingSplitChatReportID, billable`